### ISSUE TITLE
[Issue]: Movies and TV Shows Missing after Update

### ISSUE THREAD

**Issue by mike-lloyd03** [jellyfin/jellyfin#12058]
### Please describe your bug

I'm running on Arch via Docker. I upgraded to the latest image (10.9.6). After updating, I have zero Movies and TV Shows. Clicking on either library shows zero results. The files are still intact on the filesystem, however.

I see some entries like this in the log:
```
[2024-06-09 09:46:37.974 -07:00] [WRN] [59] MediaBrowser.Controller.Entities.BaseItem: Unable to find linked item at path "/media/mike/movies/The.Lord.of.the.Rings.The.Return.of.the.King.EXTENDED.2003.1080p.BrRip.x264.YIFY.mp4"
[2024-06-09 09:46:37.974 -07:00] [WRN] [59] MediaBrowser.Controller.Entities.BaseItem: Unable to find linked item at path "/media/mike/movies/The.Lord.of.the.Rings.The.Two.Towers.2002.ExD.1080p.BrRip.x264.YIFY.mp4"
```

And multiple entries like this
```
[2024-06-09 04:03:55.823 -07:00] [WRN] [27] Microsoft.EntityFrameworkCore.Model.Validation: The entity type '"AccessSchedule"' is configured to use schema '"jellyfin"', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
```

I've tried rescanning my library to no avail.

I'm not sure what docker image version I was running previouly.

### Reproduction Steps

1. docker compose pull
2. docker pull -d
3. 

### Jellyfin Version

10.9.0

### if other:

10.9.6

### Environment

```markdown
- OS: GNU/Linux
- Linux Kernel: 6.9.3-arch1-1
- Virtualization: Docker
- Clients: Web
- Browser: Chromium
- FFmpeg Version: n6.1.1 (on host, not sure of version inside container)
- Playback Method: Direct Play
- Hardware Acceleration: none
- GPU Model: none
- Plugins: AudioDB, InfuseSync, MusicBrainz, OMDb, Open Subtitles, Studio Images, TMDb
- Reverse Proxy: caddy
- Base URL: yes
- Networking: Docker bridge
- Storage: Local
```

### Jellyfin logs

```shell
[2024-06-09 04:03:55.823 -07:00] [WRN] [27] Microsoft.EntityFrameworkCore.Model.Validation: The entity type '"AccessSchedule"' is configured to use schema '"jellyfin"', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[2024-06-09 04:03:55.824 -07:00] [WRN] [27] Microsoft.EntityFrameworkCore.Model.Validation: The entity type '"ActivityLog"' is configured to use schema '"jellyfin"', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[2024-06-09 04:03:55.824 -07:00] [WRN] [27] Microsoft.EntityFrameworkCore.Model.Validation: The entity type '"CustomItemDisplayPreferences"' is configured to use schema '"jellyfin"', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[2024-06-09 04:03:55.824 -07:00] [WRN] [27] Microsoft.EntityFrameworkCore.Model.Validation: The entity type '"DisplayPreferences"' is configured to use schema '"jellyfin"', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[2024-06-09 04:03:55.824 -07:00] [WRN] [27] Microsoft.EntityFrameworkCore.Model.Validation: The entity type '"HomeSection"' is configured to use schema '"jellyfin"', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[2024-06-09 04:03:55.824 -07:00] [WRN] [27] Microsoft.EntityFrameworkCore.Model.Validation: The entity type '"ImageInfo"' is configured to use schema '"jellyfin"', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[2024-06-09 04:03:55.824 -07:00] [WRN] [27] Microsoft.EntityFrameworkCore.Model.Validation: The entity type '"ItemDisplayPreferences"' is configured to use schema '"jellyfin"', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[2024-06-09 04:03:55.824 -07:00] [WRN] [27] Microsoft.EntityFrameworkCore.Model.Validation: The entity type '"Permission"' is configured to use schema '"jellyfin"', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[2024-06-09 04:03:55.824 -07:00] [WRN] [27] Microsoft.EntityFrameworkCore.Model.Validation: The entity type '"Preference"' is configured to use schema '"jellyfin"', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[2024-06-09 04:03:55.824 -07:00] [WRN] [27] Microsoft.EntityFrameworkCore.Model.Validation: The entity type '"ApiKey"' is configured to use schema '"jellyfin"', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[2024-06-09 04:03:55.824 -07:00] [WRN] [27] Microsoft.EntityFrameworkCore.Model.Validation: The entity type '"Device"' is configured to use schema '"jellyfin"', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[2024-06-09 04:03:55.824 -07:00] [WRN] [27] Microsoft.EntityFrameworkCore.Model.Validation: The entity type '"DeviceOptions"' is configured to use schema '"jellyfin"', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[2024-06-09 04:03:55.824 -07:00] [WRN] [27] Microsoft.EntityFrameworkCore.Model.Validation: The entity type '"User"' is configured to use schema '"jellyfin"', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[2024-06-09 09:46:34.036 -07:00] [INF] [64] Emby.Server.Implementations.ScheduledTasks.TaskManager: "IntervalTrigger" fired for task: "Scan Media Library"
[2024-06-09 09:46:34.036 -07:00] [INF] [64] Emby.Server.Implementations.ScheduledTasks.TaskManager: Queuing task "RefreshMediaLibraryTask"
[2024-06-09 09:46:34.036 -07:00] [INF] [66] Emby.Server.Implementations.ScheduledTasks.TaskManager: Executing "Scan Media Library"
[2024-06-09 09:46:34.040 -07:00] [INF] [66] Emby.Server.Implementations.IO.LibraryMonitor: Stopping directory watching for path "/media/mike/music"
[2024-06-09 09:46:34.041 -07:00] [INF] [66] Emby.Server.Implementations.IO.LibraryMonitor: Stopping directory watching for path "/media/mike/movies"
[2024-06-09 09:46:34.041 -07:00] [INF] [66] Emby.Server.Implementations.IO.LibraryMonitor: Stopping directory watching for path "/media/mike/shows"
[2024-06-09 09:46:34.041 -07:00] [INF] [66] Emby.Server.Implementations.IO.LibraryMonitor: Stopping directory watching for path "/media/sherrie"
[2024-06-09 09:46:34.042 -07:00] [INF] [66] Emby.Server.Implementations.Library.LibraryManager: Validating media library
[2024-06-09 09:46:37.974 -07:00] [WRN] [59] MediaBrowser.Controller.Entities.BaseItem: Unable to find linked item at path "/media/mike/movies/The.Lord.of.the.Rings.The.Return.of.the.King.EXTENDED.2003.1080p.BrRip.x264.YIFY.mp4"
[2024-06-09 09:46:37.974 -07:00] [WRN] [59] MediaBrowser.Controller.Entities.BaseItem: Unable to find linked item at path "/media/mike/movies/The.Lord.of.the.Rings.The.Two.Towers.2002.ExD.1080p.BrRip.x264.YIFY.mp4"
[2024-06-09 09:46:37.984 -07:00] [WRN] [59] MediaBrowser.Controller.Entities.BaseItem: Unable to find linked item at path "/media/mike/movies/The.Lord.of.the.Rings.the.Fellowship.of.the.Ring.EXTENDED.2001.720p.BrRip.x264.BOKUTOX.YIFY.mp4"
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Chathula** [comment]
Same here. i can't see one tv show in the list. but I can see all other stuff

**Comment by erickwatson** [comment]
I'm in a similar situation with my music libray.
Since 10.9.5 I've been missing most of my Artists info past "D", with only one Artist showing up per Letter Category.
However, all Albums scan fine.
Example:
Searching "The Ecstatic" finds the Album, searching "Mos Def" returns no results.

**Comment by RookieZoe** [comment]
Similar issue with my music library

**Comment by PlaidShirt** [comment]
My music library mostly disappeared after upgrading the docker image to 10.9.x as well.  The fix described in Issue #11528 fixed the problem for me.  Check your docker file for the `DOTNET_SYSTEM_GLOBALIZATION_INVARIANT` variable and see if it's set to `1` or `true`.  If so, try removing it per the link to the other issue.

**Comment by erickwatson** [comment]
> My music library mostly disappeared after upgrading the docker image to 10.9.x as well. The fix described in Issue #11528 fixed the problem for me. Check your docker file for the `DOTNET_SYSTEM_GLOBALIZATION_INVARIANT` variable and see if it's set to `1` or `true`. If so, try removing it per the link to the other issue.

This is my docker file, no idea what's going on.

PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
HEALTHCHECK_URL=http://localhost:8096/health
DEBIAN_FRONTEND=noninteractive
LC_ALL=en_US.UTF-8
LANG=en_US.UTF-8
LANGUAGE=en_US:en
JELLYFIN_DATA_DIR=/config
JELLYFIN_CACHE_DIR=/cache
JELLYFIN_CONFIG_DIR=/config/config
JELLYFIN_LOG_DIR=/config/log
JELLYFIN_WEB_DIR=/jellyfin/jellyfin-web
JELLYFIN_FFMPEG=/usr/lib/jellyfin-ffmpeg/ffmpeg
XDG_CACHE_HOME=/cache
MALLOC_TRIM_THRESHOLD_=131072
NVIDIA_VISIBLE_DEVICES=all
NVIDIA_DRIVER_CAPABILITIES=compute,video,utility
TZ=Australia/Adelaide
user=1026:100

**Comment by mike-lloyd03** [comment]
I solved my issue. Somehow my parental controls got set to TV-Y7 for all users. Setting it to nothing made all my shows and movies visible again.

Must have been a bug in a database migration somewhere upgrading to the newer version.
